src/views/Login.vue

```diff
@@ -83,9 +83,7 @@ import { RestExc } from '@lokavaluto/lokapi-browser';
           login: this.email,
           password: this.password,
         });
-        this.$store.dispatch("setAccounts");
         this.success = "Connection réussie";
-        this.$store.state.lokapi.isLog = true;
```

Comment:
I'm under the impression that all usages of `isLog` are useless. Can we have one commit removing all of them in all the code in one go ?

Reply:
If you want.

Reply:
So I only kept this change in a new commit at the end of the branch.
I discarded the other change.